pyaedt/edb_core/edb_data/hfss_simulation_setup_data.py

```diff
@@ -1164,12 +1164,48 @@ def use_max_refinement(self, value):
         self._parent._update_setup()
 
     @pyaedt_function_handler()
-    def add_adaptive_frequency_data(self, frequency, max_num_passes=10, max_delta_s=0.02):
+    def add_adaptive_frequency_data(self, frequency=0, max_num_passes=10, max_delta_s=0.02):
         """Add a setup for frequency data.
 
         Parameters
         ----------
-        frequency : str, float
+        low_frequency : str, float
+            Frequency with units or float frequency (in Hz).
+        high_frequency : str, float
+            Frequency with units or float frequency (in Hz).
+        max_num_passes : int, optional
+            Maximum number of passes. The default is ``10``.
+        max_delta_s : float, optional
+            Maximum delta S. The default is ``0.02``.
+
+        Returns
+        -------
+        bool
+            ``True`` if method is successful, ``False`` otherwise.
+        """
+        low_freq_adapt_data = self._parent._edb.simsetupdata.AdaptiveFrequencyData()
+        low_freq_adapt_data.MaxDelta = self._parent._edb.edb_value(max_delta_s).ToString()
+        low_freq_adapt_data.MaxPasses = max_num_passes
+        low_freq_adapt_data.AdaptiveFrequency = self._parent._edb.edb_value(frequency).ToString()
+        high_freq_adapt_data = self._parent._edb.simsetupdata.AdaptiveFrequencyData()
+        high_freq_adapt_data.MaxDelta = self._parent._edb.edb_value(max_delta_s).ToString()
+        high_freq_adapt_data.MaxPasses = max_num_passes
+        high_freq_adapt_data.AdaptiveFrequency = self._parent._edb.edb_value(frequency).ToString()
+        self.adaptive_settings.AdaptiveFrequencyDataList.Add(low_freq_adapt_data)
+        self.adaptive_settings.AdaptiveFrequencyDataList.Add(high_freq_adapt_data)
```

Comment:
Why is the same frequency point added twice?

Reply:
@ring630 my bad, wrong copy paste